umap/static/umap/js/modules/rules.js

```diff
@@ -71,9 +71,10 @@ class Rule {
         break
       }
     }
-    if (vars.length != 2) return
+    if (vars.length !== 2) return
     this.key = vars[0]
     this.expected = vars[1]
+    // biome-ignore lint/suspicious/noGlobalIsNan: expected might not be a number.
```

Comment:
don't get this exception

Reply:
Ok, sorry, found the comment above, `Number.isNaN` excepts a number, that may be `NaN`, while `isNaN` returns false for every non number value.